py/desispec/scripts/proc_night.py

```diff
@@ -160,6 +161,9 @@ def proc_night(night=None, proc_obstypes=None, z_submit_types=None,
         psf_linking_without_fflat: bool. Default False. If set then the code
             will NOT raise an error if asked to link psfnight calibrations
             without fiberflatnight calibrations.
+        still_acquiring: bool. If True, assume more data might be coming, e.g.
```

Comment:
Please specify it is for testing purposes only

Reply:
Done